cfgov/jinja2/v1/ask-cfpb/see-all.html

```diff
@@ -11,18 +11,23 @@ <h1>
       {{ page.title }}
     </h1>
     {% if page.glossary_terms %}
-        <div class="terms">
+        <dl class="terms" itemscope itemtype="http://schema.org/DefinedTermSet">
         {% for term in page.glossary_terms %}
             <div class="term {{'' if loop.index == 1 else 'block block__border-top block__padded-top'}}"
-                 id="{{ term.anchor(page.language) }}">
-            <h2>{{ term.name(page.language) }}</h2>
-                {{ term.definition(page.language) | richtext }}
-                {% if term.answer_page_en %}
-                    <p class="term-link"><a href="{{ term.answer_page_url(page.language) }}"> {{ _('Read more') }} </a></p>
-                {% endif %}
+                 id="{{ term.anchor(page.language) }}"
+                 itemscope
+                 itemtype="http://schema.org/DefinedTerm">
+                <link itemprop="url" href="{{ page.full_url }}#{{ term.anchor(page.language) }}">
```

Comment:
I ran this through https://validator.w3.org/, and the validator shows an error with this `link` element here. If I'm reading the error right, I *think* the problem is that only `dt` and `dd` elements are allowed as children of a `div` in a `dl`. But I'm not sure that there's any other way to get that `link` in here, and that seems like a pretty harmless validation error, so I'm fine leaving it as is.

Reply:
Good catch! I tested moving it inside the `<dt>` element and that eliminated the validation errors. You might wonder if having the `url` nested inside the `name` would be a problem, but according to Google's [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool), the properties are still understood correctly. Fixed in dd68f18eec29e9291824cbbc1c222154b50c194a.